project/BuildPlugin.scala

```diff
@@ -222,27 +227,66 @@ object BuildImplementation {
          |  IO.write(file("$testDir/bloop-config/base-directory"), dir.getAbsolutePath)
          |}
          |TaskKey[Unit]("checkInstall") := {
-         |  Thread.sleep(1000) // Let's wait a little bit because of OS's IO latency
-         |  val mostRecentStamp = (bloopConfigDir.value ** "*.config").get.map(_.lastModified).min
-         |  (System.currentTimeMillis() - mostRecentStamp)
-         |  val diff = (System.currentTimeMillis() - mostRecentStamp) / 1000
-         |  if (diff <= 15) () // If it happened in the last 15 seconds, this is ok!
-         |  else sys.error("The sbt plugin didn't write any file")
+         |  val configFiles = (bloopConfigDir.value ** "*.config").get
+         |  if (configFiles.isEmpty) {
```

Comment:
This is not doing the same as the previous piece of code. I understand that forcing resources triggers compilation and makes this fail, but you're only checking if `*.config` files exist. I wonder if there's a better solution? As in, having some kind of synchronization that allows us to assert that the config files were generated in the same sbt run?

Reply:
We could have a task that remembers when the scripted test started and check that there are config files newer than that. I'm not sure that's worth the trouble, since we're completely in control of what files are already there, and it's rather unlikely that a project already has a folder named `bloop-config` with `something.config` in it, isn't it?